specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2022-11-15-preview/examples/CosmosDBDatabaseAccountPatch.json

```diff
@@ -188,6 +189,7 @@
             "enableFullTextQuery": "True"
           },
           "enableMaterializedViews": false,
+          "enableBurstCapacity": true,
```

Comment:
When the provisioningState of account is Updating, enableBurstCapacity will be returned false. However, in this example provisioningState is Succeeded